lib/util/print.js

```diff
@@ -106,4 +107,46 @@ function printTreeTraversal(node, parentNode) {
   }
 }
 
+// Prints an expression node in LaTeX
+function printLatex(node, showPlusMinus=false) {
+  // Creates the LaTeX
+  function intoTex(node, showPlusMinus=false) {
+    const originalNodeTex = node.toTex();
+
+    let outputNodeTex;
```

Comment:
I think you could just call this `nodeTex` and not have both `originalNodeTex` and `outputNodeTex` since you only seem to use one at a time - what do you think? Or did you deliberately put this here to be more readable (I could be convinced ^^)